src/renderers/common/extras/PMREMGenerator.js

```diff
@@ -468,17 +467,17 @@ class PMREMGenerator {
 
 			if ( col === 0 ) {
 
-				cubeCamera.up.set( 0, upSign[ i ], 0 );
+				cubeCamera.up.set( 0, 1, 0 );
 				cubeCamera.lookAt( forwardSign[ i ], 0, 0 );
 
 			} else if ( col === 1 ) {
 
-				cubeCamera.up.set( 0, 0, upSign[ i ] );
+				cubeCamera.up.set( 0, - 1, 0 );
```

Comment:
Did you mean `cubeCamera.up.set( 0, 0, - 1 );`?

Even so, this code block does not look correct to me. When looking up the y-axis, the "up-direction" is different than it is when looking down the y-axis.

Reply:
`cubeCamera.up.set( 0, - 1, 0 )` It was just a trick to rotate the camera 180 degrees on the y-axis while using `lookAt()`. I believe the original version used it for the same purpose with `upSign[x]`.

Reply:
https://github.com/mrdoob/three.js/blob/c4071d4332c9184a8a24ce70a37dd3f5c9769c55/src/renderers/common/extras/PMREMGenerator.js#L475-L476
You are setting the up-direction to be parallel to the look-direction. Doing so is invalid.

When looking parallel to the y-axis, the up-direction was set to either +/- z in the original PMREM generator.